docs/Features/XMLFeatures.md

```diff
@@ -0,0 +1,121 @@
+
+![Jump To Opening Tag](../images/Features/JumpToOpeningTagXML.gif)
+
+The same applies to jumping from an opening tag to the respective closing tag.
+
+![Jump To Closing Tag](../images/Features/JumpToClosingTagXML.gif)
+
+## Rename Tag
+
+To rename a single tag and its corresponding opening/closing, highlight the tag and press `F2` or use the dropdown to select `Rename Symbol`.
+
+![Rename Tag](../images/Features/RenameTagXML.gif)
+
+## Completion Support
+
+### Basic completion
+
+If an XML file is not associated to a bound grammar file, you can utilize XML completion which uses existing XML tag elements.
+
+### Completion based on grammar
+
+#### Completion based on XSD
+
+When a XML file is associated with a XSD file, there is support for completion based on the tags, attribute names and values defined in the XML Schema/XSD file.
+
+![Completion Based On XSD](../images/Features/CompletionBasedOnXSD.gif)
+
+#### Completion based on DTD
+
+When XML file is associated with a DTD file, completion based on tags, attributes name and value defined in the DTD file are supported.
+
+![Completion Based On DTD](../images/Features/CompletionBasedOnDTD.gif)
+
+## Jump to Grammar Definition
+
+If an XML file is associated with an XSD file, [vscode-xml](https://github.com/redhat-developer/vscode-xml) supports jumping from the XML tag to the schema defintion.
+
+![Jump to XSD Def From XML](../images/Features/JumpToXSDDefFromXML.gif)
+
+## Hover Support
+
+### Hover based on grammar
+
+#### Hover based on XSD
+
+When XML file is associated with a XSD file, hover documentation based on tags, attributes name and value defined in the XML Schema/XSD file are supported.
+
+![Hover Based On XSD](../images/Features/HoverBasedOnXSD.gif)
+
+#### Hover based on DTD
+
+When XML file is associated with a DTD file, hover documentation based on tags, attributes name and value defined in the DTD  file are supported.
+
+![Hover Based On DTD](../images/Features/HoverBasedOnDTD.gif)
+
+## Validation Support
+
+### Syntax validation
+
+LemMinX will show syntax errors in your XML documentation, and will provide quick fixes to resolve these errors.
+
+![Validate XML](../images/Features/ValidationXML.gif)
+
+### Read more
+
+See [XML Validation](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Validation.md#xml-validation) for more details.
+
+## Generate Grammar from XML
+
+### Generate XSD from XML
+
+Using an XML file, an XML Schema/XSD file can be generated from the opening tag in the XML file. With the cursor on the first opening tag, use `Ctrl + .` or `Quick Fix...` and select `Generate foo.xsd and bind with *` to create the file in the the same directory.
+
+![GenerateXSDFromXML](../images/Features/GenerateXSDFromXML.gif)
+
+### Generate DTD from XML
+
+See [Validation with DTD Grammar](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Validation.md#validation-with-dtd-grammar) for more.
```

Comment:
This link also needs to be changed

Reply:
Am I safe to assume when you say change, you mean do the same as you mentioned above?

Reply:
Yes